### PR TITLE
Replace string

### PR DESCRIPTION
**Pull request by otaviojava**
During the Java compilation performance the JDK does improvements with String, this way, these may be replaced with java.lang.String expressions which results in simpler and possibly more efficient code.

### CODE REVIEW

File: src/test/java/org/apache/commons/lang3/BooleanUtilsTest.java [hunk 1]
```diff
@@ -321,7 +321,7 @@ public void test_toBoolean_String() {
         assertFalse(BooleanUtils.toBoolean("false"));
         assertFalse(BooleanUtils.toBoolean("a"));
         assertTrue(BooleanUtils.toBoolean("true")); // interned handled differently
-        assertTrue(BooleanUtils.toBoolean(new StringBuffer("tr").append("ue").toString()));
+        assertTrue(BooleanUtils.toBoolean("tr" + "ue"));
```
Comment (britter) [inline src/test/java/org/apache/commons/lang3/BooleanUtilsTest.java]: Doesn't make sense at all

Comment (otaviojava) [inline src/test/java/org/apache/commons/lang3/BooleanUtilsTest.java]: It is the same test, but without the ´new `StringBuilder`

Comment (britter) [inline src/test/java/org/apache/commons/lang3/BooleanUtilsTest.java]: Yes I understand. But the old one didn't make sense as well :-) I think we can drop this assertion completely.

**Comment by britter** [comment]
@otaviojava Commons Lang has been migrated from svn to git. Because of this the main branch has been renamed to 'master'. Can you please rebase your work against the master branch? Thank you!

**Comment by chonton** [comment]
Most changes were committed in https://github.com/apache/commons-lang/commit/cc1aed9bdf196403c673a886bbf723101171a9bf

**Comment by otaviojava** [comment]
Thank you guys.
So this PR is not more necessary.

**Pull request closed without merging**